### DISCUSSION TITLE
Fill background color left to right

### DISCUSSION THREAD

**Discussion by TIEverton** [tailwindlabs/tailwindcss#12050] [Help] [answered]
How can I simulate this effect with tailwind?

https://codepen.io/kay8/pen/azKbjN

**Comment by wongjn** [comment]
You should be able to apply the same CSS properties as the example using Tailwind classes.

**Reply by TIEverton** [reply]
I believe that some of these classes are not available in tailwind as
   background: linear-gradient(to right, black 50%, white 50%);
   background-size: 200% 100%;
   background-position: right bottom;

or I don't have the slightest idea how to implement this

**Reply by wongjn** [reply, marked as answer]
They are available:
- `background-image: linear-gradient(to right, black 50%, white 50%)` can be applied with `bg-gradient-to-right from-black from-50% to-white to-50%`.
- `background-size: 200% 100%` can be applied with `bg-[length:200%_100%]`.
- `background-position: right bottom` can be applied as `bg-right-bottom`.

**Reply by filipmyllari** [reply]
@TIEverton Could you share your solution? I am currently trying to make this work in tailwind

**Reply by andymccray** [reply]
@filipmyllari I’d probably do something like this to avoid transitioning the `background-position` property: https://play.tailwindcss.com/VdwQLu9ebv
